src/labeler-yml/generate.ts

```diff
@@ -48,12 +48,12 @@ function getPatternsForTeam(
       .filter((pLine) => {
         return (
           pLine.type === "rule" &&
-          lineContainsTeamName(pLine as IInterestingCSTLine, pTeamName)
+          lineContainsTeamName(pLine as IRuleCSTLine, pTeamName)
         );
       })
       // @ts-expect-error ts thinks it can still be an IBoringCSTLine,
       // but with the filter above we've ruled that out
-      .map((pLine: IInterestingCSTLine) => pLine.filesPattern)
+      .map((pLine: IRuleCSTLine) => pLine.filesPattern)
```

Comment:
nice to  express that the labeller generator  currently only supports lines with _rules_ on them and not sections